ui/src-tauri/src/main.rs

```diff
@@ -518,15 +520,16 @@ async fn reset_account(
 
 /// Returns receiving keys to front end to display once user is logged in.
 #[tauri::command]
-async fn receiving_keys(server_store: State<'_, ServerStore>) -> Result<Vec<String>, ()> {
+async fn address(server_store: State<'_, ServerStore>) -> Result<String, ()> {
     if let Some(store) = &mut *server_store.lock().await {
-        let keys = store
-            .get_receiving_keys(Message {
+        let key = store
+            .get_address(Message {
+                // TODO: do we need to passing network?
```

Comment:
@zqhxuyuan I don't think the network matters because the address is the same for all

Reply:
It doesnt hurt us in any way honestly, keeps it readable and will probably be easier to know where to change in the future if we have a need to pass different networks